nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -0,0 +1,108 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from typing import TypeVar
+
+from copy import deepcopy
+import onnx
+
+from nncf.common.graph.model_transformer import ModelTransformer
+from nncf.common.graph.transformations.commands import TransformationCommand
+from nncf.common.quantization.structs import QuantizationMode
+
+from nncf.experimental.onnx.graph.onnx_graph import ONNXGraph
+
+from nncf.experimental.onnx.graph.transformations.layout import ONNXTransformationLayout
+
+from nncf.experimental.onnx.graph.transformations.commands import ONNXQuantizerInsertionCommand
+
+ModelType = TypeVar('ModelType')
+
+QUANTIZER_NAME_PREFIX = 'QuantizeLinear_'
```

Comment:
These constants are needed for `ONNXModelTransformer`. It would be clearer to have it inside the class.

Reply:
Make sense

Reply:
Fixed